.gitignore

```diff
@@ -34,6 +34,7 @@ test-reports
 # Ignore root testing sql & python files
 /test*.sql
 /test*.py
+/test*.txt
```

Comment:
Why this?

Reply:
Unrelated change - but when I use cProfile, I tend to output to .txt, so adding this (currently unused) namespace to give me somewhere in root to dump my outputs

Reply:
Ok - I don't have a good reason for us to use txt files for much, so makes sense.